io/src/lib.rs

```diff
@@ -89,15 +89,15 @@ impl Read for &[u8] {
 
 pub struct Cursor<T> {
     inner: T,
-    pos: u64,
+    pos: usize,
```

Comment:
I believe the reason for this was that `Cursor` was intended to be compatible with the `Seek` trait. But we shouldn't need it and should be able to remove `Cursor` entirely.

Reply:
I'm going to drop this change an stick with the `u64`.